wolfssl/wolfcrypt/wc_port.h

```diff
@@ -836,7 +836,8 @@ WOLFSSL_API int wolfCrypt_Cleanup(void);
     #endif
 #endif
 #if !defined(XGMTIME) && !defined(TIME_OVERRIDES)
-    #if defined(WOLFSSL_GMTIME) || !defined(HAVE_GMTIME_R) || defined(WOLF_C99)
+    // Always use gmtime_r if available, even when using C99 standard.
+    #if (defined(WOLFSSL_GMTIME) || defined(WOLF_C99)) && !defined(HAVE_GMTIME_R)
```

Comment:
I changed this condition to always include gmtime_r if it is available.  I wasn't clear if WOLF_C99 means "C99 standard is supported" or "only use C99".  If it's the latter, I can revert this change.